src/high-level/constructors.lisp

```diff
@@ -112,25 +112,43 @@ The tensor is specialized on TYPE with shape (floor(RANGE))."
 ;;       then the tensor will not return the correct values. Not a
 ;;       huge deal but something that be improved (similarly to how
 ;;       FROM-LIST does it.)
-(defun from-array (array shape &key type (layout :row-major))
+;; XXX: This gets the type of an array from the array, not
```

Comment:
Can you raise this as an issue for discussion

Reply:
#108